File: bitget-golang-sdk-api/pkg/model/mix/order/cancelorderreq.go
package order

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: cancel the order request
 */
type CancelOrderReq struct {
	/**
	 * Currency pair
	 */
	Symbol string `json:"symbol"`
	/**
	 * Deposit currency
	 */
	MarginCoin string `json:"marginCoin"`
	/**
	 * Order Id
	 */
	OrderId string `json:"orderId"`
}

File: bitget-node-sdk-api/src/lib/model/spot/order/CancelOrderReq.ts
export class CancelOrderReq {
    private _symbol!:string;

    private _orderId!:string;


    get symbol(): string {
        return this._symbol;
    }

    set symbol(value: string) {
        this._symbol = value;
    }

    get orderId(): string {
        return this._orderId;
    }

    set orderId(value: string) {
        this._orderId = value;
    }
}